resources/assets/js/objects/Profile.vue

```diff
@@ -81,7 +82,7 @@
             </div>
             <div class="column">
               <selector-input v-model="selectedCountry"
-                              :defaultData="selectedCountry"
+                              :defaultData="form.country"
```

Comment:
Hit error when update profile. Error in console is "The data property "countries" is already declared as a prop. Use prop default value instead.'